scripts/jenkins/Dockerfile

```diff
@@ -1,4 +1,8 @@
-FROM nvidia/cuda:10.1-devel-ubuntu18.04
+ARG BASE=nvidia/cuda:10.2-devel
```

Comment:
Above says `stage('CUDA-10.1-NVCC-DEBUG')` but this is 10.2?

Reply:
Typo